osu.Game/Scoring/ScoreImporter.cs

```diff
@@ -127,15 +135,54 @@ private void populateUserDetails(ScoreInfo model)
             if (model.RealmUser.OnlineID == APIUser.SYSTEM_USER_ID)
                 return;
 
+            if (model.RealmUser.OnlineID > 1)
+            {
+                model.User = lookupUserById(model.RealmUser.OnlineID) ?? model.User;
```

Comment:
This is not going to scale. We're doing a full user lookup with every score import? What about when someone wants to import all their scores from stable?

I think this needs to be a post process.

Reply:
I'm not sure why this is a review concern in the context of this pull given

- we're already doing this on `master`
- there is local caching in place to specifically address the "import from stable" concern
- the other existing user cache cannot be used due to reasons explained in the inline commentary added in this pull

Reply:
I didn't notice the implementation of `lookupUserByName`.